libs/facilities/frontend/view/src/lib/components/detail/detail.page.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActivatedRoute } from '@angular/router';
+
+import { XdDetailPage } from './detail.page';
+
+describe('DetailComponent', () => {
+	let component: XdDetailPage;
+	let fixture: ComponentFixture<XdDetailPage>;
+
+	beforeEach(async () => {
+		await TestBed.configureTestingModule({
+			imports: [XdDetailPage],
+			providers: [
+				{
+					provide: ActivatedRoute,
+					useValue: { snapshot: { params: { id: '1' } } },
+				},
+			],
+		}).compileComponents();
+
+		fixture = TestBed.createComponent(XdDetailPage);
+		component = fixture.componentInstance;
+		fixture.detectChanges();
+	});
+
+	it('should create', () => {
+		expect(component).toBeTruthy();
+	});
+});
```

Comment:
Where are the tests?

Reply:
There is almost nothing on this page yet. We will add a test as part of the issue https://github.com/orgs/amosproj/projects/43/views/2?pane=issue&itemId=62886932